### PR TITLE
Lesson-1-3

### PR DESCRIPTION
**Pull request by MarcusCHill**
Used react props to create todo list items and react state with callback handler to display paragraph containing user submitted title input values

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
-function AddTodoForm (){
+function AddTodoForm (props){
+
+  function handleAddTodo (e){
+    e.preventDefault();
+    let todoTitle = e.target.title.value;
+    e.target.title.value = "";
+    props.onAddTodo(todoTitle);
+    console.log(todoTitle);
```
Comment (jkylehumphrey) [inline src/AddTodoForm.js:10]: This is good for debugging, but should be removed before merging into what would be a final product.

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+  };
+
+
   return(
-    <form>
-      <label htmlFor = "todoTitle">Title: </label>
-      <input id ="todoTitle" type = "text"/>
-      <button type = "submit">Add</button>
+    <form  onSubmit={handleAddTodo}>
```
Comment (jkylehumphrey) [inline src/AddTodoForm.js:15]: Formatting is much better

**Review by jkylehumphrey — CHANGES_REQUESTED** [review]
Looks good! Just one comment

File: src/AddTodoForm.js (hunk 1 above)
Comment (jkylehumphrey) [reply src/AddTodoForm.js:10]: I looked at the lesson and see that they asked you to do this.  Sorry about that.  Nice job following directions 😀

**Review by jkylehumphrey — APPROVED** [review]

File: src/AddTodoForm.js (hunk 1 above)
Comment (MarcusCHill) [reply src/AddTodoForm.js:10]: Thank you! I totally agree with your comment and will keep that in mind for production ready code.

**Pull request merged**